app/feature/feature-addon-purchase/src/main/kotlin/com/hedvig/android/feature/addon/purchase/data/SubmitAddonPurchaseUseCase.kt

```diff
@@ -3,22 +3,32 @@ package com.hedvig.android.feature.addon.purchase.data
 import arrow.core.Either
 import arrow.core.raise.either
 import com.apollographql.apollo.ApolloClient
+import com.hedvig.android.apollo.safeExecute
 import com.hedvig.android.core.common.ErrorMessage
-import com.hedvig.android.featureflags.FeatureManager
-import kotlinx.datetime.LocalDate
+import com.hedvig.android.logger.LogPriority
+import com.hedvig.android.logger.logcat
+import octopus.UpsellTravelAddonActivateMutation
 
 internal interface SubmitAddonPurchaseUseCase {
-  suspend fun invoke(quoteId: String, addonId: String): Either<ErrorMessage, LocalDate>
+  suspend fun invoke(quoteId: String, addonId: String): Either<ErrorMessage, Unit>
```

Comment:
We just do not  have this information anymore I suppose? And show a generic success message instead?

Reply:
We do have it, but from previous parameters (from summary screen)